src/convertToObject.js

```diff
@@ -7,6 +7,24 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  const obj = {};
```

Comment:
[CODE STYLE] - Avoid variable namings like `obj`. It will decrease code readability. Consider using a more descriptive name like `styleObject` or `cssProperties`.